--- turin-cosmo-1.0.0.8.efs.json5	Fri Dec 12 01:34:06 2025
+++ turin-cosmo-1.0.0.8.mbist-mfg-v0-dext-len3-noaggr.efs.json5	Fri Dec 12 01:38:48 2025
@@ -11050,7 +11050,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistAggressorsChannelDdrMode: "All"
+                        MemMbistAggressorsChannelDdrMode: "SubChannel"
                       }
                     },
                     {
@@ -11115,7 +11115,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistPatternLengthDdr: 6
+                        MemMbistPatternLengthDdr: 3
                       }
                     },
                     {
@@ -11332,7 +11332,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistPatternSelectDdr: "Auto"
+                        MemMbistPatternSelectDdr: "Prbs"
                       }
                     },
                     {
@@ -11412,7 +11412,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistDataEyeSilentExecutionDdr: "Disabled"
+                        MemMbistDataEyeSilentExecutionDdr: "Enabled"
                       }
                     },
                     {
@@ -11502,7 +11502,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistTestModeDdr: "Both"
+                        MemMbistTestModeDdr: "DataEye"
                       }
                     },
                     {
@@ -11631,7 +11631,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistAggressorsDdr: "Auto"
+                        MemMbistAggressorsDdr: "Disabled"
                       }
                     },
                     {
