--- turin-cosmo-1.0.0.a.efs.json5	Tue Mar  3 23:55:24 2026
+++ turin-cosmo-1.0.0.a.mbist-mfg-v0-dext-len3-noaggr.efs.json5	Tue Mar  3 23:57:50 2026
@@ -11179,7 +11179,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistAggressorsChannelDdrMode: "All"
+                        MemMbistAggressorsChannelDdrMode: "SubChannel"
                       }
                     },
                     {
@@ -11244,7 +11244,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistPatternLengthDdr: 6
+                        MemMbistPatternLengthDdr: 3
                       }
                     },
                     {
@@ -11461,7 +11461,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistPatternSelectDdr: "Auto"
+                        MemMbistPatternSelectDdr: "Prbs"
                       }
                     },
                     {
@@ -11541,7 +11541,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistDataEyeSilentExecutionDdr: "Disabled"
+                        MemMbistDataEyeSilentExecutionDdr: "Enabled"
                       }
                     },
                     {
@@ -11631,7 +11631,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistTestModeDdr: "Both"
+                        MemMbistTestModeDdr: "DataEye"
                       }
                     },
                     {
@@ -11760,7 +11760,7 @@
                     },
                     {
                       Byte: {
-                        MemMbistAggressorsDdr: "Auto"
+                        MemMbistAggressorsDdr: "Disabled"
                       }
                     },
                     {
